src/lib/utilities/popup/types.ts

```diff
@@ -0,0 +1,44 @@
+// Popup Types
+// Note: these are a simple iteration based on the official docs.
+
```

Comment:
```suggestion
import type { ArrowOptions, FlipOptions, OffsetOptions, ShiftOptions } from '@floating-ui/dom';
```

Reply:
Unfortunately as discussed on Discord these are not possible to use within the library files.